### DISCUSSION TITLE
issue with FBW side stick

### DISCUSSION THREAD

**Discussion by kdullabh** [flybywiresim/aircraft#4686] [Help] [answered]
Hi, this is my first time starting a new discussion, I've tried searching for the answer on this topic with no luck.  Just wanted to see if anyone has issue with the lasted Dev build FBW the side stick is no longer working for me using the keyboard input (num pad 4 & 6 left and right) is this a bug/has anyone know a work around with this issue? I note this works on the latest stable build v0.60.  Appreciate if any can help.

**Comment by aguther** [comment, marked as answer]
https://docs.flybywiresim.com/start/autopilot-fbw/

Known issue due to a bug in the SDK
